OfficeAddin/04 Building Add-ins for Microsoft Outlook/README.md

```diff
@@ -0,0 +1,37 @@
+# Office Add-ins: Building Add-ins for Microsoft Outlook
+
+This module walks through development of mail add-ins for Microsoft Outlook.
+
+## Lab
+
+This lab will walk through the development of a message compose add-in for Microsoft Outlook using a number of developer capabilities such as ribbon commands, dialogs, function files, crafting email messages, and more.
+
+- [Lab Manual](./Lab.md)
+
+## Demos
+
+- [Mail Add-ins Basics](./Demos/01%20Mail%20Add-ins%20Basics)
+- [Add-in Commands and Dialogs](./Demos/02%20Add-in%20Commands%20and%20Dialogs)
+- [Task Pane Experiences](./Demos/03%20Task%20Pane%20Experiences)
+
+## Watch the Module
+
+This module has been recorded and is available in the Office Development YouTube channel: [Building Office Add-ins for Microsoft Outlook](https://www.youtube.com) TODO
```

Comment:
The screencasts are recorded & published once the module has been approved. The process of recording, editing & publishing a screencast is time-consuming, so we wait to do it until after the module review & any requested edits have been completed. Once you approve & merge this in, I'll record it. Then, when MSFT has published it, I'll submit a new PR updating this link to the video. This placeholder shows that a screencast is coming...

Reply:
@andrewconnell can you change the text to indicate "<TODO link to recording when available> If it will take more than a day it looks broken to document that it "is" available, and then there is an actual link, but it only goes to YouTube. Thanks!

Reply:
OK... just removed the text in the most recent commit. I'll submit a different PR when it's published.